src/Resolvers/AbstractDataResolver.php

```diff
@@ -48,4 +48,21 @@ final public function resolve(array $data, array $params = []) {
      * @return mixed
      */
     abstract protected function resolveInternal(array $data, array $params);
+
+    /**
+     * @return Schema|null
+     */
+    public function getSchema(): ?Schema {
+        return $this->schema;
+    }
+
+    /**
+     * Define groups that the hydrator belongs to.
+     * Hydrators will always belong to the root hydrate group in addition to ones defined here.
+     *
+     * @return array
```

Comment:
This could use some more explanation. Is this a `string[]`?

Reply:
Yup, fixed it.